resources/views/service/edit_registration.blade.php

```diff
@@ -0,0 +1,125 @@
+@extends('layouts.service_master')
+
+@section('title', 'Check / Update Registration')
+
+@section('content')
+
+    @include('service.partials.flash_notices')
+
+    @include('service.partials.navbar')
+
+    <div class="content check">
+        <div class="row">
+            <form action="{{ URL::route("service.edit_registration",['id' => $registration->id]) }}" method="post">
+                {{ method_field('PUT') }}
+                {!! csrf_field() !!}
+                <input type="hidden" name="registration" value="{{ $registration->id }}">
+                <div class="col">
+                    <div>
+                        <h2>CC ID:</h2>
+                        <div class="small-button-container">
+                            <input type="text" name="cc_reference" disabled value="{{ $registration->cc_reference }}">
+                            <button>
+                                <i class="fa fa-pencil" aria-hidden="true"></i>
+                            </button>
+                        </div>
+                    </div>
+                    <div>
+                        <h2>Main Carer:</h2>
+                        <div class="small-button-container">
+                            <input type="text" name="carer" disabled value="{{ $pri_carer->name }}">
+                            <button>
+                                <i class="fa fa-pencil" aria-hidden="true"></i>
+                            </button>
+                        </div>
+                    </div>
+                    <div class="other-carers-update edit">
+                        <h2>Other Collectors:</h2>
+                        <button class="edit-button">
+                            <i class="fa fa-pencil" aria-hidden="true"></i>
+                        </button>
+                        <ul>
+                            @foreach ($sec_carers as $sec_carer)
+                                <li>{{ $sec_carer->name }}</li>
+                            @endforeach
+                        </ul>
+                        <!-- V-show when edit button is clicked
+                        <div class="small-button-container">
```

Comment:
Do you want to keep this?

Reply:
As a reminder. this page will be altered as part of "dynamic"-ifying it with jquery/vure/whatever.